roles/etcd/templates/etcd.docker.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=The Etcd Server container
+After=docker.service
```

Comment:
Having been bitten once again by a docker restart, that killed all running containers, shouldn't we use `PartOf=docker.service` as well?

(The same applies to the other docker services.)